pkg/tcpip/network/ipv4/ipv4_frag_test.go

```diff
@@ -0,0 +1,330 @@
+// Copyright 2019 The gVisor Authors.
```

Comment:
The test needs to be rewritten. In general we do not do a lot of white box testing. For something like this what should be done is something like the following

=> Create stack w/ a link endpoint w/ MTU say 100 or w/e
=> Create a TCP or a UDP endpoint
=> From the link layer complete handshake/inject fragmented data packets but do not send one fragment.
=> wait for Timeout(for test we can make the timeout configurable to be lower than 30s)
=> Check that we see an ICMP TTL exceeded message.

Reply:
We found that test code for fragmentation/reassembly was already present .
tcpip/network/fragmentation/fragmentation_test.go

We moved one of the test case in this to ipv4 folder for verifying our reasssembly timeout fix. 
This is because, in order to send ICMP timeout message we need to invoke WritePacket function present in ipv4 folder